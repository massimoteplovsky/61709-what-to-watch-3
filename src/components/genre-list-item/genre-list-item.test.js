import React from "react";
import renderer from "react-test-renderer";
import GenreListItem from "./genre-list-item.jsx";

it(`<GenreListItem /> component renders correctly`, () => {
  const tree = renderer
    .create(<GenreListItem
      onChangeActiveItemIndex={() => {}}
      id={0}
      activeItemIndex={0}
      onChangeFilmGenre={() => {}}
      genre={``}
    />).toJSON();
  expect(tree).toMatchSnapshot();
});
