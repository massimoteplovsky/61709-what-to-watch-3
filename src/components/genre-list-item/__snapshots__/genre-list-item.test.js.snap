// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GenreListItem /> component renders correctly 1`] = `
<li
  className="catalog__genres-item catalog__genres-item--active"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    
  </a>
</li>
`;
